iac-extensions/arm/src/test/resources/checks/ResourceSpecificAdminAccountCheck/Microsoft.ContainerRegistry_registries.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "resources": [
+    {
+      "name": "Noncompliant: adminUserEnabled is set to true",
+      "type": "Microsoft.ContainerRegistry/registries",
+      "apiVersion": "2019-05-01",
+      "properties": {
+        "adminUserEnabled": true
+      }
+    },
+
+    {
+      "name": "Compliant: adminUserEnabled is set to false",
+      "type": "Microsoft.ContainerRegistry/registries",
+      "apiVersion": "2019-05-01",
+      "properties": {
+        "adminUserEnabled": false
+      }
+    },
+    {
+      "name": "Compliant: adminUserEnabled is to an invalid value",
+      "type": "Microsoft.ContainerRegistry/registries",
+      "apiVersion": "2019-05-01",
+      "properties": {
+        "adminUserEnabled": "true"
+      }
+    },
+    {
+      "name": "Compliant: adminUserEnabled is not set",
+      "type": "Microsoft.ContainerRegistry/registries",
+      "apiVersion": "2019-05-01",
+      "properties": {
+        "__comment": "Compliant: default value for adminUserEnabled is false"
+      }
+    }
```

Comment:
Nitpick. You can add the case when `adminUserEnabled ` is not a string literal

Reply:
Mmhh it is expecting to find a `BooleanLiteral`, I added a case with an unexpected type `StringLiteral`. Do you mean you would like another case with an unexpected type which is not a `TextTree` based element, like an `array` or an `object`?